content/en/network_performance_monitoring/guide/manage_traffic_costs_with_npm.md

```diff
@@ -0,0 +1,64 @@
+kind: guide
+---
+Traffic is expensive, especially in the cloud. Cloud providers charge differently for traffic within an availability zone, traffic between availability zones (AZs), and traffic between regions (this pricing may differ by region). Cross-regional traffic is not only the most expensive, but also the most vulnerable to errors, latency, and security threats. 
+
+Network Performance Monitoring allows you to track all of the traffic patterns described above by mapping dependencies between any tags in Datadog, including service, container, availability zone, region, datacenter, etc… This insight into your dependencies and the amount of traffic volume they produce (which is ultimately what you are billed on) can be used to monitor and optimize on your traffic-related costs. 
+
+## Datadog’s story
+
+When we began our Kubernetes migration, migrating stateless services was (expectedly) much faster and easier than migrating our stateful services (e.g. Kafka), so we migrated our stateless services first. The outcome was terabytes of cross-AZ traffic between the stateful services (all in one AZ) and stateless services (spread across the other AZs), but then our cloud bill exploded. We used our own Network Product to identify the root cause - a suboptimal migration strategy - to optimize our network (by sharding our stateful services) and ultimately saw significant reductions in our cloud provider costs.
+
+## Steps for managing traffic costs 
+
+1. To find similar issues in your own environment, you can start by grouping your traffic flows by regions, 
+    {{< img src="network_performance_monitoring/guide/manage_traffic_costs_with_npm/region.png" alt="Group flows by region">}}
+    availability zones,
+    {{< img src="network_performance_monitoring/guide/manage_traffic_costs_with_npm/availability_zone.png" alt="Group flows by availability">}}
+    and datacenters:
+    {{< img src="network_performance_monitoring/guide/manage_traffic_costs_with_npm/datacenter.png" alt="Group flows by datacenter">}}
+    An increase in your traffic bill is almost always tied to an increase in one of these types of traffic. In many cases you may want to group traffic by asymmetric search terms - meaning, you want to see the source of the traffic in terms of one tag, but want to see the destination in terms of another. This kind of asymmetric query can be used to identify costly dependencies between your on-prem datacenters and cloud regions, 
+    {{< img src="network_performance_monitoring/guide/manage_traffic_costs_with_npm/aws_account.png" alt="Identify dependencies between datacenters and cloud regions">}}
+    as well as between clouds. One particularly helpful view is to group the source of the traffic by service, and the destination across availability zones. This will help you identify particularly noisy and expensive services that are communicating across multiple availability zones, like what we found during our migration.
```

Comment:
a screenshot here would be helpful, with a caption

I'm thinking a query like: 
Source filter: AZ:1 | Source group by: service 
Destination filter: -AZ:1 | Destination group by: AZ

when sorting the table by volume, this would show the top X services communicating most from AZ:1 to other AZ's